pkg/transform/transform.go

```diff
@@ -296,6 +307,30 @@ func (r *relLinkTransformer) TransformDestination(ctx mdformatter.SourceContext,
 		}
 	}
 
+	// Non md or image relative link, so needs link to be glued.
+	if !isMDFile(relDest) && !isImgFile(relDest) && r.glueLink != "" {
+		workingDir, err := os.Getwd()
+		if err != nil {
+			return nil, err
+		}
+		// Original path before transform needs to be figured out.
+		// Case where file is in same position after transform.
+		oldRelPath := filepath.Join(r.inputDir, curRelDir, relDest)
```

Comment:
I think we do this twice or more.. wonder if we can have nice helper function for this

Reply:
I couldn't find a way to make a clean helper function for this as we do this multiple times but with some other argument each time. 😅